app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -142,7 +142,7 @@ public void NavigationTest() {
             Thread.sleep(500);
         }catch(Exception e){}
 
-        onView(withId(R.id.nav_start_activity)).perform(click());
+        /*onView(withId(R.id.nav_start_activity)).perform(click());
```

Comment:
Do you remove the start activity button ?

Reply:
Yes, I remove start activity button and General Chat button from the menu because these buttons only made sense for testing and not for real use